pwiz_tools/Skyline/TestRunnerLib/RunTests.cs

```diff
@@ -517,29 +554,55 @@ public bool Run(TestInfo test, int pass, int testNumber, string dmpDir, bool hea
             return false;
         }
 
+        public static void CleanupMSAmandaTmpFiles()
+        {
+            // MSAmanda intentionally leaves tempfiles behind (as caches in case of repeat runs)
+            // But our test system wants a clean finish
+            // TODO(MattC): tidy up MSAmanda implementation so that we can distinguish intentional uses of tmp dir (caching potentially re-used files) from accidental directory creation and/or not-reused files within
+            var msAmandaTmpDir = Path.Combine(Path.GetTempPath(), @"~SK_MSAmanda" /* must match MSAmandaSearchWrapper.MS_AMANDA_TMP */);
+            try
+            {
+                Directory.Delete(msAmandaTmpDir, true);
+            }
+            catch
+            {
+                // ignored
+            }
+        }
+
         /// <summary>
         /// Resets the test directory to empty when cleanupLevel is 'all'.
         /// </summary>
         /// <returns>A list of the entries, if any, that were removed</returns>
-        private List<string> CleanUpTestDir()
+        private List<string> CleanUpTestDir(string tmpTestDir, bool final)
         {
+            CleanupMSAmandaTmpFiles();
```

Comment:
Put this in the tests that need it or put a TODO comment on it, if these still get created all the time.

Reply:
There's already one in the function itself, FWIW, I'll add another here.